File: 2533-bitwise-xor-of-all-pairings/2533-bitwise-xor-of-all-pairings.java
class Solution {
    public int xorAllNums(int[] nums1, int[] nums2) {
        int ans =  0 ; 
        if(nums2.length%2==1){
            for(int i = 0 ; i<nums1.length;i++){
                ans ^=nums1[i] ;
            }
        }
        if(nums1.length%2==1){
            for(int i = 0; i<nums2.length;i++){
                ans ^=nums2[i] ;
            }
        }
        return ans ;
    }
}

File: 2533-bitwise-xor-of-all-pairings/2533-bitwise-xor-of-all-pairings.c
int xorAllNums(int* nums1, int nums1Size, int* nums2, int nums2Size) {
    int xor = 0 ; 
    if(nums1Size&1){
        for(int i = 0 ; i<nums2Size;i++){
            xor = xor^nums2[i] ; 
        }
    }
    if(nums2Size&1){
        for(int i = 0 ; i<nums1Size;i++){
            xor = xor^nums1[i] ; 
        }
    }
    return xor ;
}